spectroscopy/code_src/spitzer_functions.py

```diff
@@ -63,7 +63,7 @@ def SpitzerIRS_get_spec(sample_table, search_radius_arcsec, COMBINESPEC):
 
         # Now extract spectra and put all in one array
         specs = []
-        for tt in tab:
+        for tt in tab_final:
```

Comment:
This code change *does* change the results, and appears to be what the author intended. The code block above this produces `tab_final` which contains the single row from `tab` that is closest to the target if `COMBINESPEC` is True, else a full copy of `tab`. In the original code, `tab_final` was not ever used, so `COMBINESPEC` had no effect -- spectra were always combined.

Reply:
yes, I agree that the original intent of the code is to use COMBINESPEC to determine if the spectra should be combined or chosen based on the nearest, and I agree that is not happening (good catch!).  I agree this change you suggest makes sense.